tools/reactive-controllers/src/PendingState.ts

```diff
@@ -58,6 +58,7 @@ export class PendingStateController<T extends HostWithPendingState>
                       id="loader"
                       size="s"
                       indeterminate
+                      aria-label=${pendingLabel}
```

Comment:
Blocking: This change adds an `aria-label` to the progress-circle. Previously, only the host element had an `aria-label`. When the component is in `pending` state, this results in two labels being announced, causing the screen reader to read the label twice. Can you confirm?